src/components/activityTable.js

```diff
@@ -34,7 +44,7 @@ export default function ActivityTable() {
 
   useEffect(() => {
     setLoading(true)
-    getxDaiTransfers(2)
+    getxDaiTransfers(transferLimit)
       .then(
         (result) => {
           let tfrs = result.data.transfers
```

Comment:
Next time use full name variables like `transfersData` or `transfers` instead of `tfrs`